citi-onboarding/maracatronics
# Feature/base keystone

## What i did
- Create Banner model;
- Create Banner controller;
- Create Banner router.

## How to test
1. First step you have to use the command `git fetch --all` on your terminal;
2. Second, change your branch to "feature/base-keystone" using `git checkout feature/base-keystone` command;
3. After, enter on server folder with `cd server`;
4. Now, run the server with `node index.js`;
5. On your browser go to http://localhost:3001/admin
6. Click in Banner section and change the image;
7. Check if the image was changes on MongoDB.
